pandapipes/plotting/simple_plot.py

```diff
@@ -243,97 +279,71 @@ def create_simple_collections(net, respect_valves=False, respect_in_service=True
         collections["ext_grid"] = eg_coll
 
     if 'source' in net and plot_sources and len(net.source) > 0:
-        if respect_in_service:
-            source_colls = create_source_collection(
-                net, sources=net.source[net.source.in_service].index, size=source_size,
-                patch_edgecolor='silver', line_color='silver', linewidths=pipe_width)
-        else:
-            source_colls = create_source_collection(
-                net, size=source_size, patch_edgecolor='silver', line_color='silver',
-                linewidths=pipe_width)
+        idx = net.source[net.source.in_service].index if respect_in_service else net.source.index
+        source_colls = create_source_collection(net, sources=idx, size=source_size,
+                                                patch_edgecolor='silver', line_color='silver',
+                                                linewidths=pipe_width)
         collections["source"] = source_colls
 
     if 'sink' in net and plot_sinks and len(net.sink) > 0:
-        if respect_in_service:
-            sink_colls = create_sink_collection(
-                net, sinks=net.sink[net.sink.in_service].index, size=sink_size,
-                patch_edgecolor='silver', line_color='silver', linewidths=pipe_width)
-        else:
-            sink_colls = create_sink_collection(
-                net, size=sink_size, patch_edgecolor='silver', line_color='silver',
-                linewidths=pipe_width)
+        idx = net.sink[net.sink.in_service].index if respect_in_service else net.sink.index
+        sink_colls = create_sink_collection(net, sinks=idx, size=sink_size,
+                                            patch_edgecolor='silver', line_color='silver',
+                                            linewidths=pipe_width)
         collections["sink"] = sink_colls
 
     if 'valve' in net:
         valve_colls = create_valve_collection(net, size=valve_size, linewidths=pipe_width,
                                               color=valve_color, respect_valves=respect_valves)
         collections["valve"] = valve_colls
 
+    if 'flow_control' in net:
+        flow_control_colls = create_flow_control_collection(
+            net, size=flow_control_size, linewidths=pipe_width, color=flow_control_color,
+            respect_in_service=respect_in_service)
+        collections["flow_control"] = flow_control_colls
```

Comment:
Why are you changing the structure here? Why is respect_in_service part of the function, whilst in all other cases it is not?

Reply:
The implementation was more similar to the valve collections but I think it makes sense to align it with the other collections, for which in_service is considered prior to  the create...collection() calls. I've changed it.